src/components/HTMLEngineProvider/HTMLRenderers/MentionReportRenderer.tsx

```diff
@@ -63,7 +63,12 @@ function MentionReportRenderer({style, tnode, TDefaultRenderer, reports, ...defa
 
     const currentReportID = useCurrentReportID();
     const currentReport = getReport(currentReportID?.currentReportID);
-    const isGroupPolicyReport = useMemo(() => (currentReport && !isEmptyObject(currentReport) ? ReportUtils.isGroupPolicy(currentReport) : false), [currentReport]);
+
+    // When we invite someone to a room they don't have the policy type, but we still want them to be able see and click on report mentions, so we also check the reportType in case the policyType was empty.
+    const isGroupPolicyReport = useMemo(
+        () => (currentReport && !isEmptyObject(currentReport) ? ReportUtils.isGroupPolicy(currentReport) || ReportUtils.isChatRoom(currentReport) : false),
```

Comment:
Will this account for all report types on policies? Expense reports, workspace chats, transaction threads, user created rooms, default rooms, etc?

Reply:
I don't think you can invite people into Expense reports, workspace chats and transaction threads. 

So, for invited people it should account for created and default rooms, and for policy members it should account for all the ones you mentioned.

Reply:
Oh you actually can invite to the workspace chat via whispers

Reply:
Asked about [here](https://expensify.slack.com/archives/C03U7DCU4/p1713533747722469).

But thinking better about this, I believe that if a report has a policy that is not `_FAKE_` we should render the mention.

So we should be able to change this check to `currentReport.policyID && currentReport.policyID !== CONST.POLICY.ID_FAKE`

Reply:
That makes sense to me.